test_mae.py

```diff
@@ -0,0 +1,84 @@
+import torch
```

Comment:
it seems like `test_mae` and `test_huggingface_mae` are doing the same thing?

Reply:
`test_mae` just loads our last.pickle model using torch, `test_huggingface_mae` is where we load it via huggingface patterns